src/index.js

```diff
@@ -26,8 +26,14 @@ function makeStack() {
             stack.splice(index, 1);
         }
     }
+
+    function pop(e) {
```

Comment:
i'm not sure what existing handlers expect in terms of the event... definitely release this as a minor version bump (not a patch).